### ISSUE TITLE
Button-wrapped icons in NavigationBar are not vertically aligned.

### ISSUE THREAD

**Issue by thejpanganiban** [shoutem/ui#152]
`<Button>`-wrapped `<Icon>`s does not seem to render correctly. If you look at the screenshots below, they seem to move up a little bit. I've added a `<Button>`-wrapped text on the left side to see if it also affects `<Text>` elements – but it doesn't.

```
  1 import React from 'react';
  2 import { View, Text } from 'react-native';
  3 import { connect } from 'react-redux';
  4 import { NavigationBar, Icon, Title, Button } from '@shoutem/ui';
  5
  6 function HomeScene(props) {
  7   return (
  8     <NavigationBar
  9       leftComponent={<Button><Text>Hello</Text></Button>}
 10       centerComponent={<Title>Loans</Title>}
 11       rightComponent={(
 12         <Button><Icon name='cart' /></Button>
 13       )}
 14     />
 15     )
 16 };
```
<img width="371" alt="screen shot 2017-02-07 at 10 58 16 am" src="https://cloud.githubusercontent.com/assets/16813343/22676082/5d4ca746-ed24-11e6-94d1-2f43217e282c.png">

---

I'm running on:
* @shoutem/ui ^0.10.8
* react-native 0.41.1

**Comment by rsekulic1** [comment]
@thejpanganiban Thanks for submitting the issue. We're working on fix for this issue and we will release a new version of @shoutem/ui, hopefully next week, with the fix included.

**Comment by bluetidepro** [comment]
I'm having this same issue.

**Comment by zrumenjak** [comment]
Fixed with #216
